setup.py

```diff
@@ -37,5 +37,5 @@
         '': ['*.txt', '*.md', '*.rst', '*.json', '*.conf', '*.html',
              '*.css', '*.ico', '*.png', 'LICENSE', 'LEGAL', '*.indy']},
     include_package_data=True,
-    install_requires=['indy-node-dev==1.5.555'],
+    install_requires=['indy-node-dev==1.6.71', 'sovtoken==0.9.1+1.88', 'sovtokenfees==0.9.1+1.88'],
```

Comment:
This is the master branch, so dev version of sovrin, so it needs to depend on `indy-node-dev`'s version (master version). 1.6.71 is a stable version of Indy Node. So, the correct values is `1.6.585`

Reply:
These plugin packages are dependent on stable. I'll make a PR against the stable branch.